packages/tests-unit/tests/event-mapper.test.ts

```diff
@@ -110,5 +110,35 @@ describe("convertTo", () => {
         ],
       });
     });
+
+    describe("blacklisted headers", () => {
+      it("should not include keep-alive header", () => {
```

Comment:
Ideally we should probably include the full list of headers here in a single test. And add some uppercase keys as well.

Reply:
sure, done!